### PR TITLE
remove create_skill()

### PR DESCRIPTION
**Pull request by mikejgray**
Will throw errors and interfere with args and kwargs

### CODE REVIEW

**Comment by mikejgray** [comment]
re: failing test, I think the ovos packages lock us below requests 2.30...don't think we can address that at this time

**Comment by emphasize** [comment]
pip audit v1.07 is the cause (depends on 2.30)

File: __init__.py [hunk 1]
```diff
@@ -64,10 +64,3 @@ def stop(self):
         If not relevant to your skill, feel free to remove.
         """
         pass
-
-
-def create_skill():
-    """Optional for OVOSSkill, required for MycroftSkill.
-    Left for backwards compatibility reasons but it can be safely removed.
-    """
-    return HelloWorldSkill()
```
Comment (emphasize) [inline __init__.py:73]: @JarbasAl to my knowledge this will prevent using this skill as a git clone in `skills`

Comment (JarbasAl) [reply __init__.py:73]: it shouldn't, the file loader also detects the class object directly

it wont work in classic mycroft-core, but we dont care about that....

Comment (mikejgray) [reply __init__.py:73]: They have their own Hello world skill. 🙂

**Review by JarbasAl — APPROVED** [review]

**Pull request merged**